go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/alecthomas/jsonschema v0.0.0-20200530073317-71f438968921
 	github.com/celestiaorg/celestia-app v0.12.0-rc2
 	github.com/celestiaorg/go-libp2p-messenger v0.1.0
-	github.com/celestiaorg/nmt v0.12.0
+	github.com/celestiaorg/nmt v0.13.0
```

Comment:
It is a bit weird: when I run `make test-unit` with this change only (ignoring the rest below) on main, the tests get stuck after `github.com/celestiaorg/celestia-node/share/availability/discovery`.
I'm trying to find out if we changed anything unintentionally when doing the changes in nmt.